js/stores/appStore.js

```diff
@@ -328,6 +328,9 @@ function handleChangeSettingAction (settingKey, settingValue) {
 }
 
 const handleAppAction = (action) => {
+  const ledger = require('../../app/ledger')
+  ledger.boot()
```

Comment:
I'm not super familiar with the ledger but from looking at it I think I can remove the .boot() line. Because the "recover" and "backup" actions shouldn't be able to be fired unless ledger is already enabled -- right?